processing/src/main/java/org/apache/druid/query/topn/TopNNumericResultBuilder.java

```diff
@@ -150,13 +150,13 @@ public TopNNumericResultBuilder addEntry(
     }
     for (int i = extra; i < metricVals.length; i += LOOP_UNROLL_COUNT) {
       metricValues.put(aggFactoryNames[i + 0], metricVals[i + 0]);
-      metricValues.put(aggFactoryNames[i + 1], metricVals[i + 1]);
-      metricValues.put(aggFactoryNames[i + 2], metricVals[i + 2]);
-      metricValues.put(aggFactoryNames[i + 3], metricVals[i + 3]);
-      metricValues.put(aggFactoryNames[i + 4], metricVals[i + 4]);
-      metricValues.put(aggFactoryNames[i + 5], metricVals[i + 5]);
-      metricValues.put(aggFactoryNames[i + 6], metricVals[i + 6]);
-      metricValues.put(aggFactoryNames[i + 7], metricVals[i + 7]);
+      metricValues.put(aggFactoryNames[i + 1], metricVals[i + 1]); // lgtm[java/index-out-of-bounds]
```

Comment:
IMO would be nice to have a meta-comment here about why these are needed. Something like:

```
// LGTM.com flags this, but it's safe because we know "metricVals.length - extra" is a multiple of LOOP_UNROLL_COUNT.
```

Reply:
Yep, it's better. Thanks a lot.